package.json

```diff
@@ -19,6 +19,7 @@
     "build-download": "ncc build ./download/src --out ./download/dist",
     "link": "yarn run build && cd link && npm link",
     "build": "ncc build ./src",
+    "dev-build": "ncc build -M ./src",
```

Comment:
We can remove this if we want. But this is very faster compare to `build` since this doesn't minify.